librpcsegss: build failure (x86_64-musl)

```
=> librpcsecgss-0.19_4: running pre-build hook: 02-script-wrapper ...
=> librpcsecgss-0.19_4: running do_build ...
Making all in src
make[1]: Entering directory '/builddir/librpcsecgss-0.19/src'
/bin/sh ../libtool  --tag=CC   --mode=compile cc -DPACKAGE_NAME=\"librpcsecgss\" -DPACKAGE_TARNAME=\"librpcsecgss\" -DPACKAGE_VERSION=\"0.19\" -DPACKAGE_STRING=\"librpcsecgss\ 0.19\" -DPACKAGE_BUGREPORT=\"nfsv4-wg@citi.umich.edu\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DSTDC_HEADERS=1 -DHAVE_STDDEF_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_UNISTD_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_MEMSET=1 -DHAVE_STRERROR=1 -I. -I./../include/rpcsecgss -I/usr/include/gssglue   -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe    -MT librpcsecgss_la-auth_gss.lo -MD -MP -MF .deps/librpcsecgss_la-auth_gss.Tpo -c -o librpcsecgss_la-auth_gss.lo `test -f 'auth_gss.c' || echo './'`auth_gss.c
/bin/sh ../libtool  --tag=CC   --mode=compile cc -DPACKAGE_NAME=\"librpcsecgss\" -DPACKAGE_TARNAME=\"librpcsecgss\" -DPACKAGE_VERSION=\"0.19\" -DPACKAGE_STRING=\"librpcsecgss\ 0.19\" -DPACKAGE_BUGREPORT=\"nfsv4-wg@citi.umich.edu\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DSTDC_HEADERS=1 -DHAVE_STDDEF_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_UNISTD_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_MEMSET=1 -DHAVE_STRERROR=1 -I. -I./../include/rpcsecgss -I/usr/include/gssglue   -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe    -MT librpcsecgss_la-authgss_prot.lo -MD -MP -MF .deps/librpcsecgss_la-authgss_prot.Tpo -c -o librpcsecgss_la-authgss_prot.lo `test -f 'authgss_prot.c' || echo './'`authgss_prot.c
/bin/sh ../libtool  --tag=CC   --mode=compile cc -DPACKAGE_NAME=\"librpcsecgss\" -DPACKAGE_TARNAME=\"librpcsecgss\" -DPACKAGE_VERSION=\"0.19\" -DPACKAGE_STRING=\"librpcsecgss\ 0.19\" -DPACKAGE_BUGREPORT=\"nfsv4-wg@citi.umich.edu\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DSTDC_HEADERS=1 -DHAVE_STDDEF_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_UNISTD_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_MEMSET=1 -DHAVE_STRERROR=1 -I. -I./../include/rpcsecgss -I/usr/include/gssglue   -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe    -MT librpcsecgss_la-svc.lo -MD -MP -MF .deps/librpcsecgss_la-svc.Tpo -c -o librpcsecgss_la-svc.lo `test -f 'svc.c' || echo './'`svc.c
mkdir .libs
 cc -DPACKAGE_NAME=\"librpcsecgss\" -DPACKAGE_TARNAME=\"librpcsecgss\" -DPACKAGE_VERSION=\"0.19\" "-DPACKAGE_STRING=\"librpcsecgss 0.19\"" -DPACKAGE_BUGREPORT=\"nfsv4-wg@citi.umich.edu\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DSTDC_HEADERS=1 -DHAVE_STDDEF_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_UNISTD_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_MEMSET=1 -DHAVE_STRERROR=1 -I. -I./../include/rpcsecgss -I/usr/include/gssglue -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe -MT librpcsecgss_la-svc.lo -MD -MP -MF .deps/librpcsecgss_la-svc.Tpo -c svc.c  -fPIC -DPIC -o .libs/librpcsecgss_la-svc.o
In file included from svc.c:50:
/usr/include/sys/errno.h:1:2: warning: #warning redirecting incorrect #include <sys/errno.h> to <errno.h> [-Wcpp]
    1 | #warning redirecting incorrect #include <sys/errno.h> to <errno.h>
      |  ^~~~~~~
In file included from svc.c:51:
./../include/rpcsecgss/rpc/rpc.h:43:10: fatal error: rpc/types.h: No such file or directory
   43 | #include <rpc/types.h>  /* some typedefs */
      |          ^~~~~~~~~~~~~
compilation terminated.
 cc -DPACKAGE_NAME=\"librpcsecgss\" -DPACKAGE_TARNAME=\"librpcsecgss\" -DPACKAGE_VERSION=\"0.19\" "-DPACKAGE_STRING=\"librpcsecgss 0.19\"" -DPACKAGE_BUGREPORT=\"nfsv4-wg@citi.umich.edu\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DSTDC_HEADERS=1 -DHAVE_STDDEF_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_UNISTD_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_MEMSET=1 -DHAVE_STRERROR=1 -I. -I./../include/rpcsecgss -I/usr/include/gssglue -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe -MT librpcsecgss_la-auth_gss.lo -MD -MP -MF .deps/librpcsecgss_la-auth_gss.Tpo -c auth_gss.c  -fPIC -DPIC -o .libs/librpcsecgss_la-auth_gss.o
make[1]: *** [Makefile:389: librpcsecgss_la-svc.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
 cc -DPACKAGE_NAME=\"librpcsecgss\" -DPACKAGE_TARNAME=\"librpcsecgss\" -DPACKAGE_VERSION=\"0.19\" "-DPACKAGE_STRING=\"librpcsecgss 0.19\"" -DPACKAGE_BUGREPORT=\"nfsv4-wg@citi.umich.edu\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DSTDC_HEADERS=1 -DHAVE_STDDEF_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_UNISTD_H=1 -DHAVE_STDLIB_H=1 -DHAVE_MALLOC=1 -DHAVE_STDLIB_H=1 -DHAVE_REALLOC=1 -DHAVE_MEMSET=1 -DHAVE_STRERROR=1 -I. -I./../include/rpcsecgss -I/usr/include/gssglue -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe -MT librpcsecgss_la-authgss_prot.lo -MD -MP -MF .deps/librpcsecgss_la-authgss_prot.Tpo -c authgss_prot.c  -fPIC -DPIC -o .libs/librpcsecgss_la-authgss_prot.o
In file included from authgss_prot.c:43:
./../include/rpcsecgss/rpc/rpc.h:43:10: fatal error: rpc/types.h: No such file or directory
   43 | #include <rpc/types.h>  /* some typedefs */
      |          ^~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:382: librpcsecgss_la-authgss_prot.lo] Error 1
In file included from auth_gss.c:44:
./../include/rpcsecgss/rpc/rpc.h:43:10: fatal error: rpc/types.h: No such file or directory
   43 | #include <rpc/types.h>  /* some typedefs */
      |          ^~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:375: librpcsecgss_la-auth_gss.lo] Error 1
make[1]: Leaving directory '/builddir/librpcsecgss-0.19/src'
make: *** [Makefile:319: all-recursive] Error 1
=> ERROR: librpcsecgss-0.19_4: do_build: '${make_cmd} ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
=> ERROR:   in do_build() at common/build-style/gnu-configure.sh:13
```

--- a/srcpkgs/librpcsecgss/template
+++ b/srcpkgs/librpcsecgss/template
@@ -1,16 +1,24 @@
 # Template file for 'librpcsecgss'
 pkgname=librpcsecgss
 version=0.19
-revision=4
+revision=5
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libgssglue-devel"
 short_desc="Library for RPCSECGSS support"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="BSD"
+license="BSD-3-Clause"
 homepage="http://www.citi.umich.edu/projects/nfsv4/linux/"
 distfiles="$homepage/$pkgname/$pkgname-$version.tar.gz"
 checksum=0cafb86b67e5eb4c89e8abaaad9165298946bc164d258e8925fc6dc1fa913abd
+
+case "$XBPS_TARGET_LIBC" in
+	musl) broken="rpc/rpc.h header is not available on musl"
+esac
+
+post_install() {
+	vlicense COPYING
+}
 
 librpcsecgss-devel_package() {
 	depends="libgssglue-devel ${sourcepkg}>=${version}_${revision}"

